api/api.go

```diff
@@ -38,6 +40,7 @@ import (
 const (
 	cptvGlob            = "*.cptv"
 	failedUploadsFolder = "failed-uploads"
+	rebootDelaySeconds = 5
```

Comment:
Make this a `time.Duration` and then you don't need to put "seconds" in the name :)